### DISCUSSION TITLE
Beacon Probe on SV06?

### DISCUSSION THREAD

**Discussion by Heiko910** [bassamanator/Sovol-SV06-firmware#99] [Ideas]
Hey guys,
I saw a presentation on the new Beacon probe and how fast that probe can create a bed mesh.
Has anyone of you successfully implemented this new type of probe?
Lets discuss
https://beacon3d.com/product/beacon/

![probe-1-1](https://github.com/bassamanator/Sovol-SV06-firmware/assets/145156088/dd33e73a-9c37-47b3-bfae-7d123dde92f4)

**Comment by Heiko910** [comment]
here is the mount for the beacon for a SV06.
https://www.printables.com/model/459140-beacon-mount-sovol-sv06

**Comment by Heiko910** [comment]
and here is one for the SV06+
https://www.printables.com/model/624438-sv06-fitting

**Comment by bassamanator** [comment]
This pull-request is relevant: https://github.com/bassamanator/everything-sovol-sv06/pull/32

Contains notes and a guide. Not yet merged.

**Reply by bassamanator** [reply]
Please see https://github.com/bassamanator/everything-sovol-sv06/tree/mods
